### ISSUE TITLE
Brand Header user-menu does not support keyboard navigation for desktop variant

### ISSUE THREAD

**Issue by H2CK** [telekom/scale#755]
Hi,
I am using the Brand Header with the user-menu attribute. Using with mouse everything works fine, but sadly this generated user-menu for the desktop variant is not accessible by keyboard only. The mobile variant is fully accessible with keyboard only.

There is no posibility to navigate to the menu items (desktp variant) of the user-menu with tab-key or up-/down-key. 

You can verify this also directly on https://www.brand-design.telekom.com/scale/?path=/story/components-brand-header-navigation--standard&globals=locale:en

Used version of scale-components 3.0.0-beta.33.

Please fix this accessibility bug to allow using the user-menu also only with keyboard.

**Comment by nowseemee** [comment]
Hi @H2CK, many thanks for pointing out this issue. 

The story you are referring to was the first draft for the user-menu feature and should have been already removed from the examples. I'll make sure to replace it with the proper example, which is on the following link: https://www.brand-design.telekom.com/scale/?path=/story/components-flyout-menu--brand-header-user-menu&globals=locale:en

This is easier to implement and fully supports keyboard control. Let me know if you have questions regarding the above example. I'm going to keep this issue open and let you know when our Storybook is up to date. Thanks again for making me aware that this is misleading in our documentation. Have a great day!

**Comment by H2CK** [comment]
I am not sure what you want to tell me.
Should the attribute user-menu of the brand header be no longer used? And the user menu should be separatly defined for the menu-icon slot?

I tried this also as an alternativ solution. You are right, in this case the keyboard navigation works. 
BUT: Then the yet reported issue #723 also applies to me. 
 
#723 does not occur as long as you have only little static content in the default-slot of scale-app-shell. But as soon as you have a lot of content e.g. an single-spa application embedded then everything in <div slot="menu-icon"> is rendered above the header.

**Comment by nowseemee** [comment]
Sorry, you are right - the `userNavigation` attribute should be used and the keyboard navigation was indeed broken. I've published a patch: `3.0.0-beta.39`. Please let me know if this fixes the issue for you. Thanks! 🙏

**Comment by H2CK** [comment]
After updating to version 3.0.0-beta.39 it works like a charm. Thank you for your really fast fix of this accessibilty problem.
